src/main/java/com/google/cloud/tools/action/app/config/DefaultDeployConfiguration.java

```diff
@@ -54,38 +53,47 @@ private DefaultDeployConfiguration(Collection<Path> deployables, @Nullable Strin
     this.version = version;
   }
 
+  @Override
```

Comment:
These annotations don't seem necessary, since the class just implements an interface.